SU-SWS/stanford_profile_helper
# D8CORE-2942: Removed content: so that dropcap displays on Firefox

# READY FOR REVIEW

# Summary
Removed `content: "Intro text"` as it interferes with `::first-letter` targetting.

# Needed By (Date)
soon

# Urgency
Nice to have

# Steps to Test

1. Check out this branch
2. Rebuild cache
3. Verify with Firefox that dropcap styles display as expected



# Affected Projects or Products
D8Core 

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2942
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
